manifests/shared/spec/fixtures/terraform/cf.yml

```diff
@@ -7,6 +7,7 @@ terraform_outputs_cell2_subnet_id: subnet-98765432
 terraform_outputs_cell3_subnet_id: subnet-98765433
 terraform_outputs_router1_subnet_id: subnet-09876543
 terraform_outputs_router2_subnet_id: subnet-98765432
+terraform_outputs_router2_subnet_id: subnet-87654321
```

Comment:
typo: should be `router3`.

The fact that nothing failed would suggest that we have a gap in our tests.